deps/rabbitmq_amqp1_0/test/system_SUITE_data/console/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Amqp;

```

Comment:
Any particular reason not to just add this to the existing F# test program?

Reply:
Because I needed to keep an amqp 1.0 connection opened as long as I needed for exploratory testing. I found it very convenient to discover the supervisor hierarchy and to manually test rabbitmqctl command, etc. 
If I could open an amqp 1.0 connection via PerfTest I would have not written this little console app.